### DISCUSSION TITLE
Viewing old local scores shows map's total combo as 0

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#20088] [Q&A] [answered]
This doesn't happen with new scores set on the latest version of osu!lazer, but all of my old local scores show the map's total max combo as 0 if you view it through the local leaderboard (this happens with all of my local scores regardless of mods, including ones imported from stable). If you view the same score through the global leaderboard, this doesn't happen.

Viewing the score through the global leaderboard (1651 total combo):
![osu_2022-09-01_22-00-54](https://user-images.githubusercontent.com/74088315/188050320-f9455d99-1a08-495f-89b9-2dbd8d77a93a.png)

Viewing the score through the local leaderboard (0 total combo):
![osu_2022-09-01_22-01-04](https://user-images.githubusercontent.com/74088315/188050339-10d0e6a1-4b91-4edd-bbf2-8c38462b4f86.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20090
